src/Components/NavBar/index.js

```diff
@@ -337,127 +337,127 @@ export default function MiniDrawer() {
 
                   <ListItem button >
                     <ListItemIcon>
-                      {/* <img src={Calendario} alt="calendario" /> */}
+                      <img src={Calendario} alt="calendario" />
                     </ListItemIcon>
-                    <ListItemText className="item" primary="Calendário" />
+            <ListItemText className="item" primary="Calendário" />
                   </ListItem>
 
-                  <ListItem button >
+          <ListItem button >
+            <ListItemIcon>
+              <img src={Trofeu} alt="times" />
+            </ListItemIcon>
+            <ListItemText className="item" primary="Times" />
+          </ListItem>
+
+          <ListItem button >
+            <ListItemIcon>
+              <img src={Bag} alt="produtos" />
+            </ListItemIcon>
+            <ListItemText className="item" primary="Produtos" />
+          </ListItem>
+                </List>
+
+              </Grid> */}
+
+
+              {/* <div className="absoluteNavBar"> */}
+
+              <List style={{ marginTop: 30 }}>
+                <Link to={"/Perfil/" + atleticaUsername()}>
+                  <ListItem button>
+                    <ListItemIcon>
+                      <img src={Home} alt="home" />
+                    </ListItemIcon>
+                    <ListItemText className="item" primary="Home" />
+                  </ListItem>
+                </Link>
+
+                <Link to={"/Feed/" + atleticaUsername()}>
+                  <ListItem button>
+                    <ListItemIcon>
+                      <img src={Feed} alt="feed" />
+                    </ListItemIcon>
+                    <ListItemText className="item" primary="Feed" />
+                  </ListItem>
+                </Link>
+                <Link to="/Times">
+                  <ListItem button>
                     <ListItemIcon>
                       <img src={Trofeu} alt="times" />
                     </ListItemIcon>
                     <ListItemText className="item" primary="Times" />
                   </ListItem>
+                </Link>
 
-                  <ListItem button >
+                <Link to="/Produtos">
```

Comment:
Não seria:
`
<Link to=}"/Produtos/" + atleticaUsername()}>
` 
Já que agora a rota de produtos precisa do username da atlética?!